.eslintrc.js

```diff
@@ -159,6 +163,7 @@ module.exports = {
     "prettier/prettier": "error",
     quotes: ["error", "double"],
     semi: "off",
+    "import/namespace": "off",
```

Comment:
https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/namespace.md

This rule is taking most of the linting time, 33% of the overall time. 

This rule check each package usages at import time, but is not trivial to us as the non-existing methods can also be detected by `unsafe-assignment` rules. 

```
import/namespace                              | 10715.327 |    33.2%
```

Reply:
we should probably add a short comment in eslintrc to document it there as well